akka-cluster/src/main/scala/akka/cluster/ClusterReadView.scala

```diff
@@ -87,7 +85,7 @@ private[akka] class ClusterReadView(cluster: Cluster) extends Closeable {
 
           e match {
             case e: MemberEvent if e.member.address == selfAddress ⇒
-              _cachedSelf = OptionVal.Some(e.member)
+              _cachedSelf = e.member
             case _ ⇒
           }
         case s: CurrentClusterState ⇒ _state = s
```

Comment:
I think it should be updated from here also

Reply:
`CurrentClusterState` is sent once, just after `ClusterReadView` initialisation, it will not contain a member with the same unique address.
new Cluster => new ClusterReadView => ClusterReadView.eventBusListener.preStart => cluster.subscribe => ... => CurrentClusterState 

Though several tests kept refreshing the state with `ClusterReadView#refreshCurrentState`, I've removed those refresh requests as `ClusterReadView` sufficiently maintains state through `ClusterDomainEvent`s.

`akka.cluster.TransitionSpec#awaitMemberStatus` still needs to perform a periodic refresh as `Down` state for a node is not propagated to `ClusterReadView`. Should I create a ticket to propagate it? I see the propagation of `Down` to `ClusterReadView` for a node was removed in https://app.assembla.com/spaces/akka/tickets/realtime_list?ticket=3075

Reply:
it's probably true that `CurrentClusterState` is empty here but I think it would be better to handle it here anyway for the sake of not relying on implementation details of where and when the ClusterReadView is initialized

Reply:
sure, added with 5e852769d525c211ed7396d8758fb97c52be2ee6